src/System.Runtime.Extensions/src/System.Runtime.Extensions.CoreCLR.csproj

```diff
@@ -0,0 +1,90 @@
+
+  <ItemGroup>
+    <Compile Include="System\Diagnostics\Stopwatch.cs" />
+    <Compile Include="System\Runtime\Versioning\FrameworkName.cs" />
+    <Compile Include="System\Net\WebUtility.cs" />
+    <Compile Include="System\Net\Configuration\UnicodeDecodingConformance.cs" />
+    <Compile Include="System\Net\Configuration\UnicodeEncodingConformance.cs" />
+    <Compile Include="System\IO\LowLevelTextWriter.cs" />
+    <Compile Include="System\IO\LowLevelStringWriter.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\System\ArrayT.cs">
+      <Link>System\ArrayT.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\Collections\Generic\LowLevelDictionary.cs">
+      <Link>System\Collections\Generic\LowLevelDictionary.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\IO\StringBuilderCache.cs">
+      <Link>System\IO\StringBuilderCache.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
+    <Compile Include="System\Diagnostics\Stopwatch.Windows.cs" />
+    <Compile Include="$(CommonPath)\System\IO\Win32Marshal.cs">
+      <Link>System\IO\Win32Marshal.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
+      <Link>Interop\Windows\Interop.Libraries.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.Errors.cs">
+      <Link>Interop\Windows\mincore\Interop.Errors.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.FormatMessage.cs">
+      <Link>Interop\Windows\mincore\Interop.FormatMessage.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.GetLastError.cs">
+      <Link>Interop\Windows\mincore\Interop.GetLastError.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.QueryPerformanceCounter.cs">
+      <Link>Interop\Windows\mincore\Interop.QueryPerformanceCounter.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.QueryPerformanceFrequency.cs">
+      <Link>Interop\Windows\mincore\Interop.QueryPerformanceFrequency.cs</Link>
+    </Compile>
+    
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="System\Diagnostics\Stopwatch.Unix.cs" />
+    <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
+      <Link>Interop\Unix\Interop.Libraries.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\libcoreclr\Interop.QueryPerformanceCounter.cs">
+      <Link>Interop\Unix\libcoreclr\Interop.QueryPerformanceCounter.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.CoreCLR.csproj" />
+    <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
```

Comment:
For the partial facades it is generally required to compile directly against the implementations as there aren't individual packages for them and they are usually in a different type-universe (i.e. mscorlib based references and not system.runtime base references). So these P2P references are necessary and correct.